cloudygames/views.py

```diff
@@ -1,28 +1,99 @@
 from django.shortcuts import render
+from django.core import serializers
 
-from rest_framework import viewsets, generics, status
+from rest_framework import viewsets, generics, status, filters
 from rest_framework.decorators import api_view
 from rest_framework.response import Response
 
 from cloudygames.serializers import GameSerializer, GameSessionSerializer, PlayerSaveDataSerializer
 from cloudygames.models import Game, GameSession, PlayerSaveData
 
+import django_filters
+import json
+
+
+######################## Filter ############################
+
+
+class GameFilter(django_filters.FilterSet):
```

Comment:
Same comment as before, I have checked this, but both of them can not access the current user.